06_gpu_and_ml/flan_t5/finetune.py

```diff
@@ -0,0 +1,242 @@
+# # Finetuning Flan-T5
+#
```

Comment:
Can you add:

```
Example by [@anishpdalal](https://github.com/anishpdalal)
```

Similar to https://modal.com/docs/guide/ex/stable_diffusion_aitemplate